ui/app/ducks/swaps/swaps.js

```diff
@@ -322,11 +327,17 @@ export {
   swapCustomGasModalClosed,
 }
 
+export function fetchMetaSwapsQuoteRefreshTime() {
```

Comment:
This function doesn't appear to be called. Where was this needed?

It would be easier to keep track of how this value was changing if was only updated inside the controller anyway. Letting the UI set it directly would be nice to avoid.

Reply:
Good call, leftover from my previous patch!